src/logic/operate.js

```diff
@@ -0,0 +1,36 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const A = Big(Number(numberOne));
+  const B = Big(Number(numberTwo));
+  let C = 0;
+
+  if (operation === '÷' && B === 0) {
+    return C === 0
+      ? 'Indetermination'
+      : 'Infinite';
+  }
+
+  switch (operation) {
+    case '+':
+      C = A + B;
+      break;
+    case '-':
+      C = A - B;
+      break;
+    case 'x':
+      C = A * B;
+      break;
+    case '÷':
+      C = A / B;
+      break;
+    case '%':
+      C = (A * B) / 100;
```

Comment:
### Heads-Up:

When it comes to `%` I think we only need one operand.
example: click `10` then `%` = 0.01

example: `num1 = (num1 *= 0.01).toString();`

Reply:
Interesting. I appreciate it.
I'll consider your suggestion when I implement this operation in the 5th milestone.